includes/MediaWiki/Api/EditEndpoint.php

```diff
@@ -141,6 +142,31 @@ public function needsWriteAccess() {
 		return true;
 	}
 
+	/**
+	 * Returns the base for the response body
+	 * @param Status $status
+	 * @return array
+	 */
+	public function getResponseBody( Status $status ): array {
+		$response = [
+			'success' => $status->isGood(),
```

Comment:
I’m also not sure about the use of `isGood()` vs. `isOK()` here, and feel like `isOK()` might make more sense… if users of the API want to check if there were errors or not, they can look at the `errors` array, no?

Reply:
To some degree I can agree with this. But feels like we are hiding something from the API consumer, personally I would like to get notified in as many ways as possible if the request completed but might have done something fishy.

To just match the term `success` to what fits the description of these functions I'd also go with `isGood` 

```
IsGood - the operation completed and didn't have any error or warnings
isOK - whether the operation completed
```

Reply:
Hm, fair enough. I guess the success field is redundant either way – `isOK()` is (now) reflected by whether there’s an entity in the response, and `isGood()` by whether there are any errors. So maybe `isGood()` is better for the `success` field.

(I don’t think we should have two fields in the response, for good and OK – I think this confusion should stay within the extension code, not be exposed to API users as well :D )